src/DataViews/include/DataViews/SymbolLoadingState.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef DATA_VIEWS_SYMBOL_LOADING_STATE_H_
+#define DATA_VIEWS_SYMBOL_LOADING_STATE_H_
+
+#include <absl/base/attributes.h>
+
+#include <string>
+
+#include "OrbitBase/Logging.h"
+
+namespace orbit_data_views {
+
+struct SymbolLoadingState {
+  // TODO(b/202140068) remove unknown when not needed anymore
+  enum State { kUnknown, kDisabled, kDownloading, kError, kLoading, kLoaded } state;
+  SymbolLoadingState(State initial_state) : state(initial_state) {}
+
+  [[nodiscard]] std::string GetDescription() const {
+    switch (state) {
+      case kUnknown:
+        return "";
+      case kDisabled:
+        return "Disabled";
+      case kDownloading:
+        return "Downloading...";
+      case kError:
+        return "Error";
+      case kLoading:
+        return "Loading...";
+      case kLoaded:
+        return "Loaded";
+    }
+    ORBIT_UNREACHABLE();
+  }
+
+  // Returns false for the default color and true for any other color, in the latter case the
+  // parameters are changed.
+  bool GetDisplayColor(unsigned char& red, unsigned char& green, unsigned char& blue) const {
+    switch (state) {
+      case kUnknown:
+        ABSL_FALLTHROUGH_INTENDED;
```

Comment:
aren't we using the std version? `[[fallthrough]];` Also, IIRC, we don't add fallthroughs on empty cases.

Reply:
I removed the explicit fallthrough. Now I dont have to decide between the absl and std version anymore ;) 

Just fyi, we use [[fallthrough]] 13 times in the repo, ABSL_FALLTHROUGH 28 times